py3-minidb.yaml

```diff
@@ -0,0 +1,78 @@
+# Generated from https://pypi.org/project/minidb/
+package:
+  name: py3-minidb
+  version: 2.0.8
+  epoch: 1
+  description: "Simple SQLite3 store for Python objects"
+  copyright:
+    - license: ISC
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: minidb
+  import: minidb
+
+data:
+  - name: py-versions
+    items:
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '300'
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - py3-supported-python
+      - py3-supported-setuptools
+      - py3-supported-gpep517
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: b18571efe558491f692d9a8413a91772c920435e
+      repository: https://github.com/thp/minidb
+      tag: ${{package.version}}
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      runtime:
+        - py${{range.key}}-sqlite3
+        - py${{range.key}}-setuptools
+        - py${{range.key}}-gpep517
```

Comment:
I think this is are not the runtime dependencies of the project. Correct me if I'm wrong? Is there a reason to include this? 

```bash
$ uv pip show minidb
Name: minidb
Version: 2.0.8
Location: /home/user/projects/uv-project/.venv/lib/python3.11/site-packages
Requires:
Required-by:
```